### PR TITLE
fix: voiceover outline for radio buttons

### PR DESCRIPTION
**Pull request by aberikashvili**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/1640

## Description
PR fixes broken outline for radio buttons, when voiceover is on

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [N/A] Text elements follow the truncation rules
- [N/A] hover state of the element follow design spec
- [N/A] focus state of the element follow design spec
- [N/A] active state of the element follow design spec
- [N/A] selected state of the element follow design spec
- [N/A] selected hover state of the element follow design spec
- [N/A] pressed state of the element follow design spec
- [N/A] Responsiveness rules - the component has modifier classes for all breakpoints
- [N/A] Includes Compact/Cosy/Tablet design
- [N/A] RTL support
2. The code follows fundamental-styles code standards and style
- [N/A] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [N/A] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [N/A] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [N/A] `fd-reset()` mixin is applied to all elements
- [N/A] Variables are used, if some value is used more than twice.
- [N/A] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [N/A ] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build:default": "npm run style:compile && npm run style:theme && npm run style:postCSS && npm run style:assets",
     "build:prod": "cross-env NODE_ENV=production npm run build",
     "deploy": "gh-pages -d storybook-static/",
-    "lint:style": "stylelint '{src,.storybook}/**/*.(sa|sc|c)ss'",
+    "lint:style": "stylelint \"{src,.storybook}/**/*.(sa|sc|c)ss\"",
```
Comment (droshev) [inline package.json:11]: is this part of the issue or it happened with autoformatting?

**Review by droshev — CHANGES_REQUESTED** [review]
@aberikashvili is this a WIP or ready for review?
If it is WIP then the the PR title should start with `[WIP]`, if not then you need to maintain the PR a little bit more and cover the items from the template

File: package.json (hunk 1 above)
Comment (aberikashvili) [reply package.json:11]: @droshev this isn't part of issue, this is windows specific fix, I ran into the problem when I was running `npm run lint:style`

Comment (droshev) [reply package.json:11]: @aberikashvili then can you open this changes in a separate PR?

File: src/radio.scss [hunk 2]
```diff
@@ -47,6 +49,11 @@ $block: #{$fd-namespace}-radio;
 
   @extend %fd-radio-input-hidden;
 
+  &__wrapper {
```
Comment (N1XUS) [inline src/radio.scss:52]: image
Same issue repeating with a checkbox. @droshev should it be fixed as a separate issue?

Comment (aberikashvili) [reply src/radio.scss:52]: @N1XUS I think so, checkbox is different component right?

**Comment by mikerodonnell89** [comment]
Trying to find a way to do this without breaking changes, @InnaAtanasova any ideas?  Can't wrap my head around why the screenreader's focus outline would be different from the focus outline we've defined

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
I can no longer navigate the radio buttons with the Tab key.

File: stories/radio/radio.stories.js [hunk 3]
```diff
@@ -25,45 +25,57 @@ export const primary = () => `<div style="display:flex;justify-content:space-bet
         <legend class="fd-fieldset__legend">Radio Buttons Cozy Mode</legend>
         <div class="fd-form__group">
             <div class="fd-form-item">
-                <input type="radio" class="fd-radio" id="pDidh761" name="radio1" checked>
-                <label class="fd-radio__label" for="pDidh761">
-                    Field label
-                </label>
+                <div class="fd-radio__wrapper">
+                    <input type="radio" class="fd-radio" id="pDidh761" name="radio1" checked>
```
Comment (InnaAtanasova) [inline stories/radio/radio.stories.js:29]: I am not sure we are following BEM with this approach. In this case you have `fd-radio__wrapper` as a parent element for `fd-radio` which is a BEM violation. A class with `__something` must be an element of a block class

**Comment by InnaAtanasova** [comment]
Closing this PR. Feel free to open it when the comments are addressed.

**Pull request closed without merging**